config/hashids.php

```diff
@@ -44,6 +44,11 @@
             'length' => '16',
         ],
 
+        'dinnerform' => [
+            'salt' => getenv('HASHIDS_SECRET_DINNERFORM'),
```

Comment:
Ook even toevoegen aan `.env.example` en `.env.docker.example`

Reply:
As far as I can see this didn't happen.